emails/_common/watcloud-email.tsx

```diff
@@ -11,7 +11,7 @@ import { getAsset, registerAsset, WATcloudURI } from "../../utils/watcloud-uri";
 registerAsset('watcloud-logo', new WATcloudURI("watcloud://v1/sha256:393767e36d5387815c15d11c506c3c820de5db41723ffc062751673621dedb15?name=1024x512%20black%401x.png"))
 
 // Wrapper for WATcloud-themed emails
-export function WATcloudEmail({
+export function Email({
```

Comment:
```suggestion
export function WATcloudEmail({
```

Oh we should keep this as-is. I was talking about the per-email function names.